app/utils/blast-undo-stack.ts

```diff
@@ -0,0 +1,15 @@
+export function blastUndoStackToRemovePasswordFromMemory(
```

Comment:
I think the gc call should still be part of this, called at the end of this function. Otherwise pw can remain in memory longer than necessary if v8 doesn't immediately run a full gc sweep.